docs/README.md

```diff
@@ -25,52 +25,47 @@
+* [ExposureClasses](usage/nessesary_considerations/exposureclasses.md)
+* [Hibernate a Cluster](usage/shoot_settings/shoot_hibernate.md)
 * [Logging](usage/logging.md)
-* [`NodeLocalDNS` feature](usage/node-local-dns.md)
+* [`NodeLocalDNS` feature](usage/dns/node-local-dns.md)
 * [OpenIDConnect presets](usage/openidconnect-presets.md)
 * [Projects](usage/projects.md)
 * [Service Account Manager](usage/service-account-manager.md)
-* [Readiness of Shoot Worker Nodes](usage/node-readiness.md)
-* [Reversed Cluster VPN](usage/reversed-vpn-tunnel.md)
-* [Shoot Cluster Purposes](usage/shoot_purposes.md)
-* [Shoot Scheduling Profiles](usage/shoot_scheduling_profiles.md)
-* [Shoot Credentials Rotation](usage/shoot_credentials_rotation.md)
-* [Shoot Kubernetes and Operating System Versioning](usage/shoot_versions.md)
-* [Shoot `KUBERNETES_SERVICE_HOST` Environment Variable Injection](usage/shoot_kubernetes_service_host_injection.md)
-* [Shoot Networking](usage/shoot_networking.md)
-* [Shoot Maintenance](usage/shoot_maintenance.md)
-* [Shoot `ServiceAccount` Configurations](usage/shoot_serviceaccounts.md)
-* [Shoot Status](usage/shoot_status.md)
-* [Shoot Info `ConfigMap`](usage/shoot_info_configmap.md)
-* [Shoot Updates and Upgrades](usage/shoot_updates.md)
-* [Shoot HA Control Plane](usage/shoot_high_availability.md)
+* [Readiness of Shoot Worker Nodes](usage/nessesary_considerations/node-readiness.md)
+* [Shoot Cluster Purposes](usage/shoot_settings/shoot_purposes.md)
+* [Shoot Scheduling Profiles](usage/shoot_settings/shoot_scheduling_profiles.md)
+* [Shoot Credentials Rotation](usage/operating_through_annotations/shoot_credentials_rotation.md)
+* [Shoot Kubernetes and Operating System Versioning](usage/shoot_updates_and_upgrades/shoot_versions.md)
+* [Shoot `KUBERNETES_SERVICE_HOST` Environment Variable Injection](usage/nessesary_considerations/shoot_kubernetes_service_host_injection.md)
+* [Shoot Networking](usage/shoot_settings/shoot_networking.md)
+* [Shoot Maintenance](usage/shoot_settings/shoot_maintenance.md)
+* [Shoot `ServiceAccount` Configurations](usage/shoot_settings/shoot_serviceaccounts.md)
+* [Shoot Status](usage/shoot_basics/shoot_status.md)
+* [Shoot Info `ConfigMap`](usage/shoot_basics/shoot_info_configmap.md)
+* [Shoot Updates and Upgrades](usage/shoot_updates_and_upgrades/shoot_updates.md)
+* [Shoot HA Control Plane](usage/shoot_settings/shoot_high_availability.md)
 * [Shoot HA Best Practices](usage/shoot_high_availability_best_practices.md)
-* [Shoot Workers Settings](usage/shoot_workers_settings.md)
-* [Accessing Shoot Clusters](usage/shoot_access.md)
-* [Supported Kubernetes versions](usage/supported_k8s_versions.md)
-* [Tolerations](usage/tolerations.md)
-* [Trigger shoot operations](usage/shoot_operations.md)
-* [Trusted TLS certificate for shoot control planes](usage/trusted-tls-for-control-planes.md)
-* [Trusted TLS certificate for garden runtime cluster](usage/trusted-tls-for-garden-runtime.md)
```

Comment:
Move these links to the operations section, don't remove them. The same applies to all documents that were moved from `usage/` to `operations/`.